src/main.cpp

```diff
@@ -262,11 +292,9 @@ int main(int argc, char **argv) {
     exit(EXIT_SUCCESS);
   }
 
-  Keyinfo ki{};
   if (drvOptions.cryptMode != CRYPTMODE_OFF) {
     if (keyFile == "") {
-      std::string p1 = "01";
-      std::string p2 = "02";
+      std::string p1, p2;
```

Comment:
somewhere I read one should

- initialize
- write one variable per line, not more

```
std::string p1 {};
std::string p2 {};
```

Reply:
One declaration per line is good, this is part of C++ core guidelines https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rl-dcl

The default constructor of string creates an empty string, so `string s {};` is needlessly verbose.